views/fields/select_single.tt

```diff
@@ -76,7 +76,15 @@
                   data-value="[% item.label_html %]"
               >
               <label class="radio-group__label" for="[% id %]_[% item.value %]">
-                [% INCLUDE fields/sub/filter.tt value=item.label_html filter=filter %]
+                [%
+                    IF item.label_plain;
+                        INCLUDE fields/sub/filter.tt value=item.label_plain filter=text_filter
+                    ELSIF item.label_html;
+                        item.label_html;
+                    ELSE;
+                        item;
+                    END;
+                %]
```

Comment:
A few more changes needed here. It's possible that `item` will contain both `label_plain` and `label_html`. If it does, `label_html` should be used in preference for this particular part of `select_single.tt`. However there are other parts of this template that also need changing accordingly. For example, further up the file there is `data-list-text` which should always use `label_plain` and a `span` which should use `label_html` if available or otherwise HTML-encode `label_plain`.

Having thought more about the `filter` parameter, I think best to remove that entirely from `user_edit.tt` and `edit.tt` and then always filter using `INCLUDE fields/sub/filter.tt value=item.label_plain filter="html"` (here and above).

Reply:
I think the first paragraph of the comment above still needs resolving.